plugins/quant/src/functions.php

```diff
@@ -6,6 +6,14 @@
     require_once(__DIR__.'/../quant-cli.php');
 }
 
+if (!function_exists('is_plugin_active')) {
```

Comment:
```suggestion
if ( !function_exists('is_plugin_active') ) {
```

Reply:
Let's do a followup branch to sort phpcs + get CI hooked up properly to monitor ongoing.